imports/plugins/core/orders/client/templates/orders.js

```diff
@@ -89,22 +79,29 @@ const OrderHelper =  {
 
 Template.orders.onCreated(function () {
   this.state = new ReactiveDict();
+  this.orderLimits = new ReactiveDict();
   this.state.setDefault({
     orders: []
   });
+  this.orderLimits.setDefault({
+    new: 2,
```

Comment:
I wonder if we shouldn't increase this a little bit. 2 seems like very few before showing a "Show More".